test/main_test.js

```diff
@@ -24,11 +24,12 @@ suite('End to End', () => {
     ];
     let doc = await documenter({
       schemas, // schema.id + content
+      docsFolder: path.join(__dirname, 'docs'),
     });
     assert.ok(doc.tarball); // testing tarball exists
   });
 
-  test('tarball contains schemas with prefix', async function(done) {
+  test('tarball contains schemas and docs with prefix', async function(done) {
```

Comment:
I like where this is going, but we can also turn it into multiple tests as well! 

* tarball contains schemas and no docs
* tarball contains docs and no schemas
* tarball contains nothing at all!
* etc

That way when I inevitably break something in this a year from now, I'll have a better idea about what I've broken.